MachineLearningStreamlitBase/apps/streamlit_prediction_component_multiclass.py

```diff
@@ -36,9 +36,18 @@ def app():
             outline-offset: 5px;
             font-size:25px;
         }</style>
-        """, unsafe_allow_html=True) 
+        """, unsafe_allow_html=True)
     st.markdown('<div class="boxBorder"><font color="RED">Disclaimer: This predictive tool is only for research purposes</font></div>', unsafe_allow_html=True)
     st.write("## Model Perturbation Analysis")
+    st.write(
+        """
+        This predictive model utilizes parameters derived from T1w Brain MRI. 
```

Comment:
maybe here replace "parameters" by "features" (to align with machine learning terminology).

Reply:
Changed